src/components/ShareURIDialog.vue

```diff
@@ -86,6 +97,14 @@ export default {
     openQRCodeRenderer() {
       this.showQrcodeRendererDialog = true
       this.show = false
+    },
+    openInExplorer() {
+      const explorerUri = config[this.crypto.toLowerCase()].explorerAddress.replace(
+        /\${([a-zA-Z_]+?)}/,
+        this.address
+      )
+      const explorerLink = this.isADM ? websiteUriToOnion(explorerUri) : explorerUri
+      window.open(explorerLink, '_blank', 'resizable,scrollbars,status,noopener')
```

Comment:
You can use [getExplorerAddressUrl](https://github.com/Adamant-im/adamant-im/blob/eacc2d3fd8c6a3a14dd532f5d560783f4d56a967/src/config/utils/getExplorerAddressUrl.ts) helper:

1. No need in `websiteUriToOnion`. Tor has a separate [config](https://github.com/Adamant-im/adamant-im/blob/658efd2b068473745615f6dfe4e4b097e62c4c27/src/config/tor.json#L51) which is automatically replaced when building the app.
2. Use [isErc20](https://github.com/Adamant-im/adamant-im/blob/e0a8fdb2980e6e85013726826f9d33dea3f0a668/src/lib/constants/index.js#L80) helper to detect ETH based crypto.

Reply:
We build the app for Tor using [tor.json](https://github.com/Adamant-im/adamant-im/blob/eacc2d3fd8c6a3a14dd532f5d560783f4d56a967/src/config/tor.json#L30) config, so the `getExplorerAddressUrl()` will choose the right URL automatically.